Specs/ASTMapSpec.m

```diff
@@ -95,9 +99,27 @@ - (void)testEverything {
             XCTAssertEqualObjects(after, (@{
                 @"fr": @"BONJOUR",
                 @"en": @"HELLO"
-                                          }));
+            }));
         }];
 
+        [XCTContext runActivityNamed:@"should return an array" block:^(id<XCTActivity> _Nonnull activity){
```

Comment:
I guess this shouldn't go into the `for arrays` scope? Either dictionary or its own is fine I guess.

Reply:
It is in the for `for dictionaries` scope currently. Am I missing something?

Reply:
My bad!